nixos/modules/services/networking/xray.nix

```diff
@@ -52,6 +53,17 @@ with lib;
           See <https://www.v2fly.org/en_US/config/overview.html>.
         '';
       };
+
+      loadCredential = mkOption {
```

Comment:
Why does this need to variable? Can't we hardcode this?

Reply:
@SuperSandro2000 What do you mean by hardcode here?